ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -108,18 +161,26 @@ const triggerSet: TriggerSet<Data> = {
       regex: /Kothornos Kick/,
       beforeSeconds: 5,
       infoText: (data, _matches, output) => {
-        if (data.role === 'tank' || data.role === 'healer')
-          return;
-        return output.text!();
+        let jumpDir;
+        if (data.jumpDir1 === 'east')
```

Comment:
Subjectively, I think the bait call comes out late.  It seems like the directions are reversed as well.

https://www.twitch.tv/videos/1268415201?t=2h23m24s

Reply:
I see where the jump directions go wrong. The value stored in jumpDir1 is the same as the starting direction, so it has to start out flipped. The ternary operator is working based on it being undefined at start to determine the direction as well.

I don't think we can really add more time outside of .3 seconds as that is when you would have to be in the thorn stack otherwise. However, it would be possible we add more time for the first kick and have a shorter time for 2nd.